tests/integration/testdata/remote_invoke/template-lambda-response-streaming-fns.yaml

```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Description: >
+  Testing application for lambda functions with response streaming
+  
+Resources:
+  NodeStreamingFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./lambda-fns/src/
+      Handler: index.handler
+      Runtime: nodejs18.x
+      Architectures:
+        - x86_64
```

Comment:
Do we need `Architectures` defined here?

Reply:
Nope, this is not required here. Removed it with the latest changes.